drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -349,15 +351,23 @@ function DragAndDropTemplates(configuration) {
         return h(
             popupSelector, {style: {display: have_messages ? 'block' : 'none'} },
             [
-                h('div.close.icon-remove-sign.fa-times-circle'),
+                h(
+                    'button.unbutton.close.icon-remove-sign.fa-times-circle',
```

Comment:
I am not accustomed to seeing a button done this way. I suggest you follow the pattern here:

http://ux.edx.org/components/buttons/
and
http://ux.edx.org/design_elements/icons/

So there would be a button containing 2 spans-- one that has the icon (always aria-hidden), and one with the sr text. Similar to this--

![image](https://cloud.githubusercontent.com/assets/484484/17782764/17e5780e-6543-11e6-809f-c746418d1392.png)